coc/raid.py

```diff
@@ -427,14 +427,15 @@ class RaidLogEntry:
                  "_response_retry"
                  )
 
-    def __init__(self, *, data, client, **kwargs):
+    def __init__(self, *, data, client, clan_tag, **kwargs):
```

Comment:
this is a breaking change, we need to move clan_tag into kwargs and pop it out from there if you really need that

Reply:
It is needed for the cache 🥹

Reply:
Yeah but you can not add it as required argument without a major release. 
It needs at least a default value

Reply:
The only public activity of this github account  #closedsource xD

Reply:
lol i responded from the wrong account🙈